src/components/ElementSearchDialog/element-search-dialog.js

```diff
@@ -11,7 +11,7 @@ import PropTypes from 'prop-types';
 import SearchIcon from '@mui/icons-material/Search';
 import { useIntl } from 'react-intl';
 
-const TERM_MIN_SIZE_BEFORE_SEARCH = 3;
+const TERM_MIN_SIZE_BEFORE_SEARCH = 1;
```

Comment:
Why do you keep this constant ?
I think we can delete it

Reply:
well i tried to removed but we can't because when we search with one character and removed there is request to the server with empty term

Reply:
you need only test if the term is empty or not

Reply:
done